radicle-httpd/src/api/v1/projects.rs

```diff
@@ -399,6 +407,89 @@ async fn issues_handler(
     Ok::<_, Error>(Json(issues))
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct IssueCreate {
+    pub title: String,
+    pub description: String,
+    pub tags: Vec<Tag>,
+}
+
+/// Create a new issue.
+/// `POST /projects/:project/issues`
+async fn issue_creation_handler(
+    State(ctx): State<Context>,
+    AuthBearer(token): AuthBearer,
+    Path(project): Path<Id>,
+    Json(issue): Json<IssueCreate>,
+) -> impl IntoResponse {
+    let sessions = ctx.sessions.write().await;
```

Comment:
```suggestion
    let sessions = ctx.sessions.read().await;
```
Should be enough? Is the presence of that token automatically mean it's authorized? Don't remember what we landed on.

Reply:
Yeah that's right, will change this.

Reply:
done in 4cde666